subprojects/packagefiles/libccp4c/meson.build

```diff
@@ -0,0 +1,104 @@
+# Meson file for libccp4c (CCP4 core libraries, C part only)
+project('libccp4c', ['c'],
+        version: '6.5.1',
+        meson_version: '>=0.49.0',
+        license: 'LGPL3')
+
+
+cc = meson.get_compiler('c')
+mdep = cc.find_library('m', required: true)
```

Comment:
This should be `required: false` to make it portable. Most platforms (in fact everything except Linux) have math functions in `libc` and don't have a standalone `m` library at all.

Also, `true` is the default value so this would be superfluous in any case.

Reply:
Ok, fixed!